wayland-protocols/src/lib.rs

```diff
@@ -7,27 +7,41 @@
 //! protocol. The creation of these modules (and the dependency on the associated crate) is
 //! controlled by the two cargo features `client` and `server`.
 //!
-//! The cargo feature `unstable_protocols` adds an `unstable` module, containing bindings
-//! to protocols that are not yet considered stable. As such, no stability guarantee is
-//! given for these protocols.
+//! ## Protocol categories
 //!
-//! Some protocols require unstable rust features, the inclusion of them is controlled
-//! by the cargo feature `nightly`.
+//! The protocols provided in this crate are grouped in 3 main categories:
+//!
+//! - The [`wp`] module contains general purpose wayland protocols
```

Comment:
Aren't these plumbing and not general?

Reply:
Well, the wayland-protocols governance document states:

> The "wp" namespace is established for protocols generally useful to Wayland implementations (i.e. "plumbing" protocols).

So, that seems rather correct to me to call them "general purpose", no?